src/components/WrapperFavorites.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+
+import Title from './Title';
+import Container from './Container';
+import Slider from './Slider';
+import SmallInfoCard from './SmallInfoCard';
+
+import api from '../service/api';
+
+type MovieId = {
+  id: number;
+}
+
+const WrapperFavorites = () => {
+  const [movieInfo, setMovieInfo] = useState<MovieId[]>([]);
+
+  useEffect(() => {
+    api.getDataByIds(api.getFavoritsIdList()).then((res: any) => {
+      setMovieInfo(res);
+    });
+  }, []);
+
+  return (!movieInfo.length ? null : (
```

Comment:
it is nicer to have check 
if(!movieInfo.length) {
return null;
}

return (markup)